morphir-ir/shared/src/main/scala/zio/morphir/ir/ValueModule.scala

```diff
@@ -1,7 +1,7 @@
 package zio.morphir.ir
 
 import zio.morphir.ir.{Literal => Lit}
-import zio.morphir.ir.TypeModule.Type
+import zio.morphir.ir.Type.Type.{unit => UnitType}
```

Comment:
This should be `Unit`, no?

Reply:
the original is unit. If you're referring to the alias I gave it, I wanted to differentiate it from the Value unit